sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_models.py

```diff
@@ -277,6 +297,15 @@ def _from_generated(cls, batch_status_details):
             has_completed=bool(batch_status_details.status not in ["NotStarted", "Running", "Cancelling"])
         )
 
+    def __repr__(self):
+        # pylint: disable=line-too-long
+        return "JobStatusResult(id={}, created_on={}, last_updated_on={}, status={}, error={}, documents_total_count={}, documents_failed_count={}, documents_succeeded_count={}, documents_in_progress_count={}, documents_not_yet_started_count={}, documents_cancelled_count={}, total_characters_charged={}, has_completed={})" \
```

Comment:
Going to approve and merge! For the future you might try breaking on the string like this rather than disabling the linting error: https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py#L373

Reply:
i did
did you open this again?

Reply:
I saw your changes. No, I'm just trying to get the pipeline to pass on the PR. It's timing out at the moment, eng sys folks are looking into it.